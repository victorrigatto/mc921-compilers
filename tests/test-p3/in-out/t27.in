int main(){
   int i, count, number[25];

   print("How many elements are u going to enter (max=25)?");
   read("value");

   print("Enter", count, "elements: ");
   for(i=0;i<count;i=i+1)
      read(number[i]);

   return 0;
}
