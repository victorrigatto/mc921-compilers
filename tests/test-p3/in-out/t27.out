SemanticError: Constant(rawtype=string,
         value=value,
         uc_type=) is not a variable @ 5:9
